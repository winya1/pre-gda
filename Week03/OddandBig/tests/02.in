8
11 13 15
15 15 15
2147483647 99 100501
9 7 5
8 6 4
10 12 14
-11 -13 -15
0 0 0
