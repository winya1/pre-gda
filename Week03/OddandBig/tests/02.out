Case #1: Yes
Case #2: Yes
Case #3: Yes
Case #4: No
Case #5: No
Case #6: No
Case #7: No
Case #8: No
